src/client/project/view_spec.py

```diff
@@ -114,28 +113,26 @@ class Assets(BaseTolokaObject):
 
 
 class TemplateBuilderViewSpec(ViewSpec, spec_value=ViewSpec.TEMPLATE_BUILDER):
-    """A template builder view specification that defines an interface with
-    template builder components
+    """A task interface defined with the [TemplateBuilder](toloka.client.project.template_builder.TemplateBuilder.md).
+
+    See also [Template Builder](https://toloka.ai/en/docs/template-builder/) in the guide.
 
     Attributes:
-        view:
-        plugins:
-        vars:
-        core_version: Default template components version. Most likely you do not need to change this parameter.
-        infer_data_spec: You can configure the data specification automatically or manually. You can change the way the
-            specification is configured using the infer_data_spec option:
+        view: A top level component like [SideBySideLayoutV1](toloka.client.project.template_builder.layouts.SideBySideLayoutV1.md).
+        plugins: An array of plugins.
+        vars: Reusable data. It is referenced with the [RefComponent](toloka.client.project.template_builder.base.RefComponent.md).
+        core_version: The default template components version. Most likely you do not need to change this parameter.
```

Comment:
`Most likely, you do not need to change this parameter.`

Reply:
Thanks.